packages/kit/test/apps/basics/src/routes/routing/prefetched/hash-route.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+	import { onMount } from 'svelte';
+	import { page } from '$app/stores';
+
+	const modalContents = [
+		{
+			id: 'please-dont-show-me',
+			title: 'Oopsie'
+		}
+	];
+
+	let modal = undefined;
+
+	const checkIfModalShouldBeShown = () => {
+		console.log('mount');
```

Comment:
my bad. I forgot to remove this debug line before I pushed the example. I would remove it

Reply:
Hmm, I'll update the test to check (and just for assertion in the future), but I don't believe it'll be an issue. `load` can potentially run on both the client and the server, but the only implementation I changed here is the client one. I would expect the client `load` to rerun.

Reply:
Right, prefetch only happens on the client 👍